src/PerfView/UserCommands.cs

```diff
@@ -1512,6 +1533,33 @@ private TraceEvents GetTraceEventsWithProcessFilter(ETLDataFile etlFile)
             return events;
         }
 
+        /// <summary>
+        /// Save the CPU stacks for an ETL file into a perfView.xml.zip file.
+        /// </summary>
+        /// <param name="etlFile">The ETL file to save.</param>
+        /// <param name="process">The process to save. If null, save all processes.</param>
+        /// <param name="warmSymbolLookupMinimumValue"> The number of samples needed to warrant looking up the symbols on the sample server. </param>
+        private static void SaveCPUStacksForProcessAsCsv(ETLDataFile etlFile, TraceProcess process = null, int warmSymbolLookupMinimumValue = 10)
+        {
+            // Focus on a particular process if the user asked for it via command line args.
+            if (process != null)
+            {
+                etlFile.SetFilterProcess(process);
+            }
+
+            var stacks = etlFile.CPUStacks();
+            stacks.Filter = new FilterParams();
```

Comment:
I don't believe that you should need to assign a `FilterParams` object if you're not setting any parameters.

Reply:
Addresed